dataset/dictionary/characters.json5

```
@@ -4761,6 +4761,177 @@
     zhCN: "欧洛伦",
     tags: [ "natlan", "character-main" ],
   },
+  {
+    en: "Pacal",
+    ja: "パカル",
+    zhCN: "帕加尔",
+    notes: "「こだまの子」族長",
+    notesZh: "「回声之子」首领",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Wayna",
+    ja: "ワイナ",
+    zhCN: "瓦伊纳",
+    notes: "「懸木の民」族長",
+    notesZh: "「悬木人」首领",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Balam",
+    ja: "バラム",
+    zhCN: "帕兰",
+    notes: "「帰火聖夜の巡礼」大会運営者",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Lamanai",
+    ja: "ラマナイ",
+    zhCN: "拉马奈",
+    notes: "「帰火聖夜の巡礼」大会運営者",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Tonina",
+    ja: "チュニーナ",
+    zhCN: "图尼娜",
+    notes: "「帰火聖夜の巡礼」大会運営者",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Atea",
+    ja: "アテア",
+    zhCN: "阿伽娅",
+    notes: "第五章第一幕に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Chuychu",
+    ja: "クイク",
+    zhCN: "葵可",
+    notes: "チャスカの義妹",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Vichama",
+    ja: "ヴィチャマ",
+    zhCN: "维查玛",
+    notes: "第五章第二幕に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Mallko",
+    ja: "マルコ",
+    zhCN: "穆尔科",
+    notes: "第五章第二幕に登場する人物",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Comonqui",
+    ja: "コモンキ",
+    zhCN: "科莫奎",
+    notes: "第五章第二幕に登場するアビス教団の詠唱者",
```

I made the note a bit more detailed.

```suggestion
    notes: "第五章第二幕に登場するアビスの詠唱者・淵炎",
```